src/index.js

```diff
@@ -33,9 +34,24 @@ const display = () => {
   const span = document.querySelectorAll('.text');
   span.forEach((btn, index) => {
     btn.addEventListener('keyup', () => {
-      const test = script.retrieve();
-      test[index].description = btn.value;
-      script.save(test);
+      const tick = script.retrieve();
+      tick[index].description = btn.value;
+      script.save(tick);
+    });
+  });
+
+  // check box
+
+  const checkbox = document.querySelectorAll('.checkbox');
+  checkbox.forEach((btn, index) => {
+    btn.addEventListener('change', () => {
+      const tick = script.retrieve();
+      if (btn.checked === true) {
+        tick[index].completed = completed(tick);
+      } else {
+        tick[index].completed = unCompleted(tick);
+      }
+      script.save(tick);
```

Comment:
- [ ] Upon user interaction, it's expected that when the checkbox is checked, the todo should be marked as completed and automatically updated in localStorage and when the page reloads it's still marked as completed until otherwise. In your implementation, it's not updating the localStorage and when the browser is reloaded, the check is disappeared. Kindly work on this. 👍🏽

<img width="1125" alt="Screenshot 2023-05-12 at 9 40 47 PM" src="https://github.com/techmoves/todo-list/assets/50941074/f1f84f9c-2d83-4295-bd0c-cce771ff3d6e">

**Kindly Observe the screenshot below**

<img width="1440" alt="Screenshot 2023-05-12 at 9 49 48 PM" src="https://github.com/techmoves/todo-list/assets/50941074/871563d1-ea03-4215-9bc3-e8418b17d7c4">